### PR TITLE
PB-1241: remove akhet library and upgrade lark_parser

### PR DESCRIPTION
**Pull request by ltclm**
remove the akhet library and revert some of the changes  
introduced by https://github.com/geoadmin/mf-chsdi3/pull/4024

static assets will be served by apache again

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -50,7 +51,8 @@ Sphinx = "~=5.2"
 sphinx-rtd-theme = "~=1.0"
 nose2 = "*"
 coverage = "~=4.5"
-numpy = "*"
+numpy = "==2.0.2"
+pyproj = "==3.6.1"
```
[the remark this answers is not in the document]
Comment (ltclm) [reply Pipfile:55]: the issue here is if it is not redundant in `dev-packages` and `packages` the latest version will be installed and the installation fails. i did not manage to pin the version without this redundancy. the version 3.7 is only working with python 3.10.

File: apache/wsgi-py3.conf.in [hunk 2]
```diff
@@ -26,13 +26,32 @@ WSGIPassAuthorization On
...
 RewriteRule ^${APACHE_ENTRY_PATH}/configs/(de|fr|it|rm|en)/translations\.json ${APACHE_ENTRY_PATH}/rest/services/translations?lang=$1 [PT]
 RewriteRule ^${APACHE_ENTRY_PATH}/configs/(de|fr|it|rm|en)/catalog\.(\w+)\.json ${APACHE_ENTRY_PATH}/rest/services/$2/CatalogServer?lang=$1 [PT]
 RewriteRule ^${APACHE_ENTRY_PATH}/configs/(de|fr|it|rm|en)/services\.json ${APACHE_ENTRY_PATH}/rest/services [PT]
 RewriteRule ^${APACHE_ENTRY_PATH}/configs/services\.json ${APACHE_ENTRY_PATH}/rest/services [PT]
 
+# Robots static files
+RewriteRule ^${APACHE_ENTRY_PATH}/robots.txt ${APACHE_ENTRY_PATH}/static/${ROBOTS_FILE} [PT]
+<LocationMatch ^${APACHE_ENTRY_PATH}/static/(robots.txt|robots_prod.txt)>
+    Header set Content-type "text/plain"
+</LocationMatch>
```
[the remark this answers is not in the document]
Comment (ltclm) [reply apache/wsgi-py3.conf.in:53]: thank you! will do that

[the remark this answers is not in the document]
Comment (ltclm) [reply apache/wsgi-py3.conf.in]: done

File: apache/wsgi-py3.conf.in [hunk 3]
```diff
@@ -26,13 +26,32 @@ WSGIPassAuthorization On
...
+RewriteRule ^${APACHE_ENTRY_PATH}/static/info\.json         https://%{HTTP_HOST}%{REQUEST_URI} [R,L]
+
+# Info
+RewriteRule ^${APACHE_ENTRY_PATH}/info.json ${APACHE_ENTRY_PATH}/static/info.json [PT]
+
+# Static for cross domain flash/arcgis
+RewriteRule ^${APACHE_ENTRY_PATH}/(crossdomain.xml|clientaccesspolicy.xml) ${APACHE_ENTRY_PATH}/static/$1 [PT]
+<LocationMatch ^${APACHE_ENTRY_PATH}/static/(crossdomain.xml|clientaccesspolicy.xml)>
+    Header set Content-type "text/x-cross-domain-policy"
+</LocationMatch>
```
[the remark this answers is not in the document]
Comment (ltclm) [reply apache/wsgi-py3.conf.in]: done

File: Pipfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (ltclm) [reply Pipfile]: with the merge of lark_parser upgrade into this pr,  the redundancy could be removed

**Review by ltshb — APPROVED** [review]

**Pull request merged**